seed/data_importer/meters_parser.py

```diff
@@ -465,7 +465,12 @@ class with the data.
 
         results = []
         for raw_reading in raw_data:
-            start_date = datetime.strptime(raw_reading['Month'], '%b-%y')
+            # December 16, 2022 updated code to match ESPM formatting changes
+            try:
+                start_date = datetime.strptime(raw_reading['Month'], '%b-%y')
+            except KeyError:
+                start_date = datetime.strptime(raw_reading['Month '], '%b-%y')
```

Comment:
ideally we strip the row headers, but this works for now. I bet they are going to fix this one and we will be back to `Month`.

Reply:
Let me know if you want me to adjust to strip the row headers.